File: app/src/main/java/com/example/livestreamingagora/models/LogoutResponse.java
package com.example.livestreamingagora.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LogoutResponse {
    @SerializedName("status")
    @Expose
    public Boolean status;
    @SerializedName("message")
    @Expose
    public String message;
    @SerializedName("data")
    @Expose
    public Object data;
    @SerializedName("errors")
    @Expose
    public Object errors;


    public LogoutResponse(Boolean status, String message, Object data, Object errors) {
        this.status = status;
        this.message = message;
        this.data = data;
        this.errors = errors;
    }

    public Boolean getStatus() {
        return status;
    }

    public String getMessage() {
        return message;
    }

    public Object getData() {
        return data;
    }

    public Object getErrors() {
        return errors;
    }

    @Override
    public String toString() {
        return "LogoutResponse{" +
                "status=" + status +
                ", message='" + message + '\'' +
                ", data=" + data +
                ", errors=" + errors +
                '}';
    }
}

File: app/src/main/java/com/example/livestreamingagora/models/LogoutResponse.kt
package com.example.livestreamingagora.models

import com.google.gson.annotations.SerializedName

class LogoutResponse(
    @field:SerializedName("status")
    val status: Boolean? = null,
    @field:SerializedName("message")
    val message: String? = null,
    @field:SerializedName("data")
    val data: String? = null,
    @field:SerializedName("errors")
    val errors: String? = null
)
